lib/content-services/src/lib/tag/services/tag.service.ts

```diff
@@ -105,6 +105,24 @@ export class TagService {
         return observableRemove;
     }
 
+    /**
+     * Deletes a tag with tagId.
+     * This will cause the tag to be removed from all nodes.
+     * You must have admin rights to delete a tag.
+     *
+     * @param tagId of the tag to be deleted
+     * @returns Null object when the operation completes
+     */
+    deleteTag(tagId: string): Observable<any> {
```

Comment:
Small thing but maybe we could be a bit more accurate with the return type here. We know this endpoint doesn't return any data so let's have it return `Observable<void>` instead. I'll let it slide for the js-api one (returning `Promise<any>` also) since that's done throughout the js-api, but we could take this opportunity to make this method in ADF a bit clearer.

Reply:
Thanks, changed to void